redis-persistence/src/main/java/com/netflix/conductor/dao/dynomite/RedisExecutionDAO.java

```diff
@@ -525,6 +529,13 @@ private String insertOrUpdateWorkflow(Workflow workflow, boolean update) {
 
 	}
 
+    private void mapTaskToWorkflow(String taskId, String workflowInstanceId) {
```

Comment:
Missing java doc.  This is not just a map operation, the correlation between the workflow and the tasks contained are updated in dynomite, consider changing the name of the method.  Adding a unit test whenever a new method is introduced is a good practice.

Reply:
added javadoc and test